doc/nrf/releases_and_maturity/releases/release-notes-changelog.rst

```diff
@@ -1058,6 +1065,9 @@ This section provides detailed lists of changes by :ref:`script <scripts>`.
 
 * Added the :file:`thingy91x_dfu.py` script in the :file:`scripts/west_commands` folder.
   The script adds the west commands ``west thingy91x-dfu`` and ``west thingy91x-reset`` for convenient use of the serial recovery functionality.
+* Added support for format version ``1`` of :file:`dfu_application.zip` file to :ref:`nrf_desktop_config_channel_script` Python script.
+  Since NCS 2.7, the script supports both format version ``0`` and format version ``1`` of the zip archive.
```

Comment:
```suggestion
* Added support for format version ``1`` of the :file:`dfu_application.zip` file to :ref:`nrf_desktop_config_channel_script` Python script.
  Since |NCS| v2.7.0 release, the script supports both format version ``0`` and ``1`` of the zip archive.
```
Additionally, organize again like this:

* Added:

   * The :file:...
   * Support for...